go/arrow/flight/flightsql/driver/driver_test.go

```diff
@@ -619,6 +619,7 @@ func (s *SqlTestSuite) TestRowsPrematureCloseDuringNextLoop() {
 	require.NoError(t, err)
 	require.Equal(t, int64(rowCount), insertedRows)
 
+	time.Sleep(200 * time.Millisecond)
```

Comment:
What's this for?

Reply:
Looks like the flaky test was that every so often sqlite didn't fully update itself internally and the select would return 0 rows instead of 6000. Likely some sort of race condition

Reply:
That...seems like a serious bug if true. More likely I'd say a missed COMMIT or just a race condition where we aren't actually finishing inserting data?

Reply:
oh well, this driver code from the start was just duplicating adbc for some reason anyways so I'm not too concerned with it